src/components/new-request/search.vue

```diff
@@ -664,6 +765,10 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
 <style lang="scss" scoped>
 @import '@/assets/styles/theme.scss';
 
+.contact-registries .v-icon.v-icon {
```

Comment:
I'd be tempted to select the icons directly, like this:
```
.v-icon.mdi-phone,
.v-icon.mdi-email {
 ...
}
```

Reply:
This looks fine now, but another way is to add `color="primary"` to the v-icon.

(The primary color is assigned in plugins/vuetify.ts, and is the same value as $app-blue.)